### PR TITLE
History, profile page

### PR DESCRIPTION
**Pull request by emmanuel-lbs**
profile page dont have anymore the loading page USER DONT EXIST now just load normaly

restrcuturaiton des matchs history du back 

du clean dans le back

### CODE REVIEW

File: back/src/user/user.controller.ts [hunk 1]
```diff
@@ -95,8 +89,9 @@ export class UserController {
   // Retrieves all match requests for a specific user
   @Post('matches')
   @UseGuards(JwtGuard)
-  GetMatches(@GetUser() user: User) {
-    return this.userService.GetMatchRequest(user.id);
+  async GetMatches(@Body('userId') userId: number) {
+    console.log("userId");
```
Comment (Blodispic) [inline back/src/user/user.controller.ts:93]: log

Comment (emmanuel-lbs) [reply back/src/user/user.controller.ts:93]: log

Comment (KobeLaC) [reply back/src/user/user.controller.ts]: loh

File: front/src/components/Chat/AdminCommands.tsx [hunk 2]
```diff
@@ -113,7 +113,7 @@ export function ConfigureChannelPrivate(props: { trigger: boolean, setTrigger: (
 
 	const AddPeoplePrivate = () => {
 		if (allfriend.length > 0)
-			socket.emit('AddPeoplePrivate', { chanid: props.channel.id, users: allfriend });
+			socket.emit('AddPeoplePrivate', { chanId: props.channel.id, users: allfriend.map(user => user.id) });
```
Comment (se-lee) [inline front/src/components/Chat/AdminCommands.tsx:116]: 'chanId' needs to be 'chanid' to match invite-user-dto
(we can fix it later with the chat-back PR )

File: back/src/user/user.controller.ts (hunk 1 above)
Comment (se-lee) [reply back/src/user/user.controller.ts]: log

**Review by KobeLaC — APPROVED** [review]
Didn't look in detail the result reduction, i believe in Jojo, others things look good

**Review by Blodispic — APPROVED** [review]
Tout me parait bon du coup ! :)

**Pull request merged**